### PR TITLE
KOGITO-8071: Flaky test - serverless-workflow-editor-extension-autocompletion.test.ts

### PR DESCRIPTION
**Pull request by zdrapela**
**Jira:** https://issues.redhat.com/browse/KOGITO-8071

The test is now stable when running locally (30+ runs, no fail) and [on the CI](https://github.com/zdrapela/kie-tools/actions/runs/3637751535) (7+ runs, no fail).

The requirement to run smoke tests before other tests was added.

### CODE REVIEW

**Review by tomasdavidorg — APPROVED** [review]

File: packages/vscode-extension-serverless-workflow-editor/it-tests/serverless-workflow-editor-extension-autocompletion.test.ts [hunk 1]
```diff
@@ -14,16 +14,17 @@
  * limitations under the License.
  */
 
+require("./serverless-workflow-editor-extension-smoke.test");
```
Comment (paulovmr) [inline packages/vscode-extension-serverless-workflow-editor/it-tests/serverless-workflow-editor-extension-autocompletion.test.ts:17]: Why are these `require`s needed?

**Review by paulovmr — CHANGES_REQUESTED** [review]
Hi @zdrapela , just one question regarding the added requires.

File: packages/vscode-extension-serverless-workflow-editor/it-tests/serverless-workflow-editor-extension-autocompletion.test.ts (hunk 1 above)
Comment (zdrapela) [reply packages/vscode-extension-serverless-workflow-editor/it-tests/serverless-workflow-editor-extension-autocompletion.test.ts:17]: It is not relevant to fixing this test, but @domhanak advised adding this to these tests. To my understanding, this should ensure, that smoke test is run before running any other tests. We use the same approach in [`vscode-extension-pack-kogito-kie-editors`](https://github.com/kiegroup/kie-tools/tree/main/packages/vscode-extension-pack-kogito-kie-editors/it-tests).

Comment (zdrapela) [reply packages/vscode-extension-serverless-workflow-editor/it-tests/serverless-workflow-editor-extension-autocompletion.test.ts:17]: Hi @paulovmr, is the explanation above enough, or should I remove the `require`s for now and we can discuss their usage later, possibly in separate PR? 🙂

**Review by paulovmr — APPROVED** [review]

**Pull request merged**